### ISSUE TITLE
Column selection extends over wrapped parts of lines

### ISSUE THREAD

**Issue by PhiLhoSoft** [atom/atom#10234]
According to the thread https://discuss.atom.io/t/vertical-column-selection/15913/7 we can do a column selection. On Windows, I can use Ctrl-Alt-Up / Down do extend the selection vertically.
This is nice and useful. But alas, this is a bit ruined when line wrapping is active (the default on Atom, I think).
I think this selection extension should skip over the wrapped part of the long lines: it makes no sense to extend the selection to these parts, as they are random (depend on the width of the editor!) and not aligned with anything.

**Comment by lee-dohm** [comment]
> line wrapping is active (the default on Atom, I think).

Soft Wrap is not enabled by default, no.

As to the rest of the idea, I think your suggested behavior would be counterintuitive. Can you cite any other examples where text or other data would be inside a selection but not selected?

**Comment by PhiLhoSoft** [comment]
Perhaps it is default only in Markdown documents, where the wrapping is the most useful, or perhaps I just played with settings so much I don't recall anything... :smile: 
Anyway, that's not relevant to the issue, but that's for correcting.

OK, I made screen captures with LICEcap to illustrate my point, with a semi-realistic use case.
One with SciTE, my favorite lightweight / "dumb" editor that I use for everyday tasks.
![columnselectionwithwrapping-scite](https://cloud.githubusercontent.com/assets/1087902/12064217/98f57b9c-afbd-11e5-8130-9a8fa721705a.gif)
Another with Atom, my currently favorite "smart" editor with IDE-like features... :wink: 
![columnselectionwithwrapping-atom](https://cloud.githubusercontent.com/assets/1087902/12064222/b5d07744-afbd-11e5-8212-9ec30b297fde.gif)

In SciTE, the selection skips the wrapped parts, although I have to hit Alt+Shift+Down to go over these parts. Why not, that's what it does in normal caret moves. Anyway, as expected, I select only the start of the lines, the common part between them.

In Atom, the wrapped part is selected too, so the rectangular selection is unusable.

I grant you, this is not a common use case, I can toggle the wrapping mode before doing my selection, etc. But still, that's in getting right small details that we recognize great editors... :wink:
